runtime/moonbeam/tests/integration_test.rs

```diff
@@ -774,7 +775,7 @@ fn claim_via_precompile() {
 
 			// Alice uses the crowdloan precompile to claim through the EVM
 			let gas_limit = 100000u64;
-			let gas_price: U256 = 1_000_000_000.into();
+			let gas_price: U256 = 100_000_000_000u64.into();
```

Comment:
Why did that change ?

Reply:
https://github.com/PureStake/moonbeam/blob/jeremy-moonbeam-phase-2-filter/runtime/moonbeam/src/lib.rs#L346
Minimum gas price is 1 GWei * SUPPLY_FACTOR (100)